src/elm/Cambiatus/Object/Claim.elm

```diff
@@ -71,6 +71,6 @@ id =
     Object.selectionForField "Int" "id" [] Decode.int
 
 
-isVerified : SelectionSet Bool Cambiatus.Object.Claim
-isVerified =
-    Object.selectionForField "Bool" "isVerified" [] Decode.bool
+status : SelectionSet String Cambiatus.Object.Claim
+status =
+    Object.selectionForField "String" "status" [] Decode.string
```

Comment:
Would it be possible to decode `status` into a custom type instead? I believe they're known at compile-time, so we could get assistance from the compiler in this case

Reply:
It is, requires a update in the backend, done here: https://github.com/cambiatus/backend/pull/42